protocol/protobuf/pubkey.go

```diff
@@ -1,11 +1,11 @@
-package applicationmetadata
+package protobuf
 
 import (
 	"crypto/ecdsa"
-	"github.com/status-im/status-go/eth-node/crypto"
+	"github.com/ethereum/go-ethereum/crypto"
```

Comment:
Ideally we would keep geth away from status-protocol dependencies. What's the reason for going back to using `ethereum/go-ethereum/crypto`?

Reply:
I think I have missed it in the rebase, well spotted